### ISSUE TITLE
Offline Fixation detetor exception

### ISSUE THREAD

**Issue by mkassner** [pupil-labs/pupil#900]
```python
Traceback (most recent call last):
  File "/Users/mkassner/Pupil/pupil_code/pupil_src/launchables/player.py", line 440, in player
    p.recent_events(events)
  File "/Users/mkassner/Pupil/pupil_code/pupil_src/shared_modules/fixation_detector.py", line 343, in recent_events
    self.correlate_and_publish()
  File "/Users/mkassner/Pupil/pupil_code/pupil_src/shared_modules/fixation_detector.py", line 400, in correlate_and_publish
    fixations_by_frame[f['start_frame_index']].append(f)
IndexError: list index out of range
```

**Comment by papr** [comment]
I was able to reproduce this issue on current `master`. This is not the case when using #904

**Comment by papr** [comment]
Fixed with #912
